src/stylesheets/_fonts.scss

```diff
@@ -39,3 +39,10 @@
     url("~fonts/montserrat-v15-latin-500italic.ttf") format("truetype"),
     url("~fonts/montserrat-v15-latin-500italic.svg#Montserrat") format("svg"); /* Legacy iOS */
 }
+
+/* source-code-pro-regular */
+@font-face {
+  font-family: "Source Code Pro";
+  font-weight: 400;
+  src: url("https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap");
+}
```

Comment:
Let's wait to push this in until we have Priya's PR in. She has the entire fonts stores locally so we can get rid of this once that PR is in.